parser.js

```diff
@@ -72,7 +73,9 @@ function parseGraph(graph, options)
 		options = {};
 	}
 
-	let createContext = options.createContext || false;
+	let namespaceContext = options.namespaceContext || false; 
+
+  let createContext = options.createContext || namespaceContext;
```

Comment:
if options.createContext is false but namespaceContext is true could there be a problem?